doc/source/examples/dem/small-scale-rotating-drum-post-processing/small-scale-rotating-drum-post-processing.rst

```diff
@@ -0,0 +1,216 @@
+
+``read_lethe_to_pyvista`` takes the name of the ``.pvd`` file generated by the simulation. The method can take 3 other arguments: ``first``, ``last``, and ``interval``, standing for the first and last time-steps to be read and the interval between the time-steps, respectively. By default, ``first = 0``, ``interval = 1``, and ``last`` is the last time-step of the data.
+
+The ``read_lethe_to_pyvista`` reading function assigns the datasets of each time-step to the object ``particles``. Each time-step corresponds to a `PyVista dataset <https://docs.pyvista.org/user-guide/vtk_to_pyvista.html#>`_, and can be accessed using ``particles.df[$TIME-step_NUMBER]``.
+
+.. note:: 
+  
+  If we set ``first = 2``, ``particles.df[0]`` corresponds to the dataset of the 2nd time-step of the simulation.
+
+
+Creation of a new array
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To color the particles according to their position, we use the function ``array_modifier``, which takes the following arguments:
+
+- ``reference_array_name``: Name of the array used to sort the data. By default: ``"ID"``
+- ``array_name``: Name of the new array. If there is an array with the same name, it will be rewritten according to the other arguments. By default: ``"new_array"``
+- ``restart_array``: If ``True``, zeroes the entire array before modifying it. If one wants to modify part of the array keeping the rest intact, it must be set as ``False``. By default: ``False``
+- ``condition``: Takes a string and uses it in an if condition to modify the array. Variables accepted include ``x``, ``y``, ``z``, ``u``, ``v``, ``w``, ``t``, and any other array (``ID`` for example). It also accepts a combination of them, such as ``"x*w**2 + t > 2 and ID > 0"``. By default: ``""``
+- ``array_values``: New values to the array. This argument accepts a single value (which will be repeated to all data respecting the given ``condition``), a `NumPy <https://numpy.org/>`_ array, or `Python list <https://docs.python.org/3/tutorial/datastructures.html>`_ (with the same length (``len``) of all other arrays in ``particles.df``), or a string such as ``"2*x + t"`` (working just like the ``condition`` argument). By default: ``0``
+- ``standard_value``: If ``restart array = True`` or the array is a completely new array, the ``standard_value`` will be plugged to the entire array before modifying it. By default: ``0``
+- ``time_dependent``: ``array_modifier`` can be time dependent or not. If set ``True``, ``condition`` will be tested to each of the time-steps, while if it is ``False``, it will be applied using the ``reference_time_step`` instead, and the modification will be just replicated to the other time-steps. By default: ``False``
+- ``reference_time_step``: Reference time-step to which the modification will be applied. If ``time_dependent = False``, the result of the modificaition applied to ``reference_time_step`` will be simply replicated to the others. By default: ``0``
```

Comment:
This parameter is so powerful!! :D 
As already said in this PR, I think you should explain that this is not really the time step, but the output step.